relayer/relayer.go

```diff
@@ -427,7 +429,7 @@ func (r *Relayer) RelayMessage(warpLogInfo *vmtypes.WarpLogInfo, storeProcessedH
 	}
 
 	// Create and run the message relayer to attempt to deliver the message to the destination chain
-	messageRelayer := newMessageRelayer(r, unsignedMessage, destinationBlockchainID)
+	messageRelayer, err := newMessageRelayer(r, unsignedMessage, destinationBlockchainID)
```

Comment:
Looks like this `err` check below was unnecessary previously. Glad it's corrected now 👍